apps/studio/src/features/editing-experience/components/LinkEditorDrawer.tsx

```diff
@@ -200,7 +200,7 @@ export const LinkEditorDrawer = () => {
     title,
   }
   const handleChange = (data: IsomerLinkSchema) =>
-    setLinkAtom((oldData) => ({ ...oldData, ...data }))
+    setLinkAtom(data as CollectionLinkProps)
```

Comment:
note: the `oldData` causes the `date` field to be preserved if it was later removed, which causes the preview to be incorrect.